tests/docker/tests_integration/run_tests.sh

```diff
@@ -29,7 +29,3 @@ dart bin/main.dart -m production -r maintenance --apply-migrations
 echo "### Running tests"
 dart test test_integration -x integration --concurrency=1
 checkLastExitCode
-
-cd ../serverpod_test_module/serverpod_test_module_server/
```

Comment:
Removed the "custom" integration test for the test module server, which can now be handled through the general CI setup which would work for all new modules created from the template (as those contain a `docker-compose` file).